core/lib/zksync_core/src/eth_sender/eth_tx_manager.rs

```diff
@@ -348,6 +416,9 @@ impl EthTxManager {
         // Not confirmed transactions, ordered by nonce
         for tx in inflight_txs {
```

Comment:
wouldn't it make sense to only load the relevant inflight_txs from the DB?

Reply:
Correct, however I wanted to save a dash of time by not introducing new queries.